production/grafanacloud-install.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
+# shellcheck shell=dash
```

Comment:
Did you mean bash?
```suggestion
# shellcheck shell=bash
```

Reply:
`dash` is a strict posix compliant shell. It's not bash compatible so it's a good target for trying to write portable scripts

Reply:
Gotcha. I'm a bit lost since this one uses `dash` and the others use `bash`. If `dash` is intentional, can we add a comment explaining it?

Reply:
Yup. I took it from the rustup script which also uses `/usr/bin/env sh`.

Reply:
Ahhh, it's because this is using `/usr/bin/env sh` and users use `bash`. Sorry 🙃 A comment would be useful here IMO